bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -201,7 +210,6 @@ public void ksEventReceived(KSEvent ksEvent) {
         if (!processing) {
             isSTTServerAborting = false;
             if (ksEvent instanceof KSpottedEvent) {
-                toggleProcessing(true);
```

Comment:
Please also remove in this case the two calls to toggleProcessing with false in the 2 catch in the code just after. They are now more needed.